index.js

```diff
@@ -2341,25 +2341,24 @@
   //. ```
   S.lastIndexOf = sanctifyIndexOf('lastIndexOf');
 
-  //# pluck :: Accessible a => TypeRep b -> String -> [a] -> [Maybe b]
+  //# pluck :: Accessible a => (b -> Boolean) -> String -> [a] -> [Maybe c]
```

Comment:
> Perhaps a `Predicate a` pseudotype?

I'm not sure how this would work. Could you provide the type signature you have in mind?